05-typedb-cluster/02-kubernetes-deployment.md

```diff
@@ -176,6 +176,8 @@ Configurable settings for Helm package include:
 | `storage.persistent`| `true`  | Whether TypeDB Cluster should use a persistent volume to store data                       |
 | `singlePodPerNode`  | `true`  | Whether TypeDB Cluster pods should be scheduled to different Kubernetes nodes             |
 | `exposed`           | `false` | Whether TypeDB Cluster supports connections via public IP (outside of Kubernetes network) |
+| `logging.enabled`   | `false` | Whether TypeDB Cluster pushes logs into Logstash                                          |
```

Comment:
why is the config called `logging` and not `logstash`?

Reply:
@lolski good point! addressed in vaticle/typedb-cluster#369 and https://github.com/vaticle/docs/pull/556/commits/14cfa782935f3da40e6baa9947ceb6e18f3d4025